playwright-e2e/tests/smoke-test.spec.ts

```diff
@@ -27,39 +30,72 @@ test("Smoke Test @smoke-test", async ({
   await createCase.submitCase(createCase.generatedCaseName);
   await createCase.checkCaseIsCreated(createCase.generatedCaseName);
 
+
   // Orders and directions sought
-  await ordersAndDirectionSought.ordersAndDirectionsNeeded();
-  await startApplication.addApplicationDetailsHeading.isVisible();
+  // await ordersAndDirectionSought.ordersAndDirectionsNeeded();
+  // await startApplication.addApplicationDetailsHeading.isVisible();
 
   // Hearing urgency
-  await startApplication.hearingUrgencyLink.isVisible();
-  await startApplication.hearingUrgencyLink.click();
-  await hearingUrgency.whenDoYouNeedHearingRadio("Within 18 days");
-  await hearingUrgency.whatTypeOfHearingDoYouNeed("Standard case management");
-  await hearingUrgency.giveReasonTextBoxFill();
-  await hearingUrgency.withoutNoticeHearing("No");
-  await hearingUrgency.needAHearingWithReducedNoise("No");
-  await hearingUrgency.respondentsAwareOfProceedings("No");
-  await hearingUrgency.continueButton.click();
-  await hearingUrgency.checkYourAnswers.isVisible();
-  await hearingUrgency.saveAndContinueButton.click();
-  await startApplication.addApplicationDetailsHeading.isVisible();
+  // await startApplication.hearingUrgencyLink.isVisible();
```

Comment:
Hi @Braimah101 is there a reason for these lines to be commented out of the smoke test?

Reply:
True